sdks/python/apache_beam/coders/coders.py

```diff
@@ -78,14 +78,16 @@
 
 # pylint: disable=wrong-import-order, wrong-import-position
 # Avoid dependencies on the full SDK.
-try:
-  # Import dill from the pickler module to make sure our monkey-patching of dill
-  # occurs.
-  from apache_beam.internal.dill_pickler import dill
-except ImportError:
-  # We fall back to using the stock dill library in tests that don't use the
-  # full Python SDK.
-  import dill
+
```

Comment:
Let's try to keep this file as is and include dill as a test dependency.

Reply:
But this file uses a method called `maybe_dill_dumps` which uses dill. Would that create an issue?